directjngine/src/test/java/com/softwarementors/extjs/djn/jscodegen/CodeFileGeneratorTest.java

```diff
@@ -0,0 +1,34 @@
+package com.softwarementors.extjs.djn.jscodegen;
+
+import org.testng.annotations.Test;
+
+import java.lang.reflect.Method;
+
+import static org.testng.Assert.*;
+
+public class CodeFileGeneratorTest {
+
+    private String getDebugFileName(String file) {
```

Comment:
I'd just make `getDebugFileName` package protected (no modifier) instead of using reflection to invoke for testing.

Reply:
I was confused because I didn't know the test rules of the project, but it's good that I can use protected function. 👍 
Applied in 68d3969